### PR TITLE
remaining TO DO for IMC: analyte_class / segment_data_format

### PR DESCRIPTION
**Pull request by cebriggs7135**
IMC: 
analyte_class

### CODE REVIEW

File: src/ingest_validation_tools/table-schemas/level-2/imc.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ fields:
   name: analyte_class
   constraints:
     enum:
-      - TODO
+      - imaging
```
Comment (cebriggs7135) [inline src/ingest_validation_tools/table-schemas/level-2/imc.yaml:18]: proteins

File: src/ingest_validation_tools/table-schemas/level-2/imc.yaml [hunk 2]
```diff
@@ -99,8 +99,7 @@ fields:
     enum:
       - um
 - name: segment_data_format
-  description: This refers to the data type, which is a float for the IMC counts.
-  type: number
+  description: This refers to the data type, which is a "Float" for the IMC counts.
```
Comment (ngehlenborg) [inline src/ingest_validation_tools/table-schemas/level-2/imc.yaml:102]: @cebriggs7135 It looks like you removed the `type` field. It was `number` but now there is no longer a type for `segment_data_format`.

Comment (cebriggs7135) [reply src/ingest_validation_tools/table-schemas/level-2/imc.yaml:102]: @ngehlenborg Yes because it actually is the word "Float", not a float number.

Comment (mccalluc) [reply src/ingest_validation_tools/table-schemas/level-2/imc.yaml:102]: @cebriggs7135 :
- I'll make this an enum: `float`, `integer`, `string` should cover it.

File: src/ingest_validation_tools/table-schemas/level-2/imc.yaml (hunk 1 above)
Comment (mccalluc) [reply src/ingest_validation_tools/table-schemas/level-2/imc.yaml:18]: add `proteins` as an option? Will do.

**Pull request merged**